### ISSUE TITLE
Quorum sensing molecules

### ISSUE THREAD

**Issue by bebi78** [OpenSourceMycetoma/Adjuvant-Screen#4]
Quorum sensing molecules as possible antifungals.

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi @MFernflower  Farnesol was identified as a quorum sensing molecule in Candida fungi, and improved the efficacy of azole treatment (see link). Are any quorum sensing mechanisms known for M. mycetomatis and other eumycetoma strains? I wonder how the eumycetoma fungi ´´decide´´ to build grains, and if farnesol might be a useful combination drug. 

https://www.mdpi.com/2309-608X/8/8/783

**Comment by MFernflower** [comment]
Farnesol is a perfume chemical no?

What happend with previous perfume chemical screen? @wwjvdsande @bebi78

**Comment by MFernflower** [comment]
Maybe most potent fenarimol + most potent perfume chemical could be a good
combo?

**Comment by bebi78** [comment]
@MFernflower Yes, farnesol is also part of essential oils and perfumes/cosmetics. It is possible that other terpene-based perfumes work similarly. As far as I know the perfume tests need to be repeated because of some testing issues with these compounds last year.

**Comment by wwjvdsande** [comment]
@bebi78  @MFernflower I expect that quorem sensing indeed plays a role. I had bought farnesol but also other quorum sensing molecules during the summer. The idea is that we will experiment during the winter to see if it influences the fungus in vitro.

**Comment by bebi78** [comment]
@wwjvdsande @MFernflower That's good. I made a quick Pubmed search, and cinnamaldehyde, eugenol and its derivatives, thymol, vanillin and bisabolene can have an effect on quorum sensing in various bacteria, where QS research is quite advanced.

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi @MFernflower Among the compounds of our naphthazarin paper, the naphthoquinones juglone, plumbagin, and lawsone, and the anthraquinone emodin were also described as quorum sensing modulators in various bacteria. I couldn't find articles about naphthazarin, but I think it might be a QS active compound, too.

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Niclosamides also possess some quorum sensing modifying properties in bacteria, and affect fungal biofilm and invasion. 
https://www.nature.com/articles/s41598-018-29973-8

**Comment by MFernflower** [comment]
Much like the tricyclics - niclosamide likely has many targets @bebi78

**Comment by bebi78** [comment]
@MFernflower Yes, there are cancer researchers who think it is a ´´magic bullet´´.

https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2022.1004978/full#:~:text=Niclosamide%20has%20anti-tumor%20effect%20through%20inhibiting%20multiple%20oncogenic,suppressor%20signaling%20such%20as%20p53%2C%20PP2A%20and%20AMPK.

**Comment by MFernflower** [comment]
I would not call niclosamid a magic bullet - much like the tricyclics it
has many targets known and unknown - using either of these classes to treat
fungus would be more like taking a hammer to biology!

It is very interesting to use so called "dirty drugs" as a feature not a
flaw

@bebi78

**Comment by bebi78** [comment]
@wwjvdsande @MFernflower Because of the relevance of farnesol I wondered if farnesyl transferase inhibitors are active. Indeed, two inhibitors, lonafarnib and tipifarnib, were tested in M. m. and F. s. as part of the Response Boxes. While tipifarnib was inactive, lonafarnib was active against M. m., but also inactive against F. s.

**Comment by MFernflower** [comment]
@bebi78 @wwjvdsande  do we know if nitroxoline was tested? It also stops biofilms from
forming!

**Comment by bebi78** [comment]
@MFernflower Seems that only its ruthenium complex was tested so far. But I suppose that the metal-free nitroxoline will be more active. I have received quite interesting antiparasitic activities for nitroxoline from my Parasitology experts last summer (unpublished).

**Comment by MFernflower** [comment]
Yes 8 hydroxyquinolines seem to be quite powerful ---> see https://youtu.be/1mHt5ucaMdI
 @bebi78

**Comment by MFernflower** [comment]
@bebi78 I don't know if perhaps you can source some of these analogues of nitroxoline but they could be something to follow up on?

![image](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/assets/3164942/1ac38c65-4048-4971-b2ac-0b83a635637f)

**Comment by bebi78** [comment]
@MFernflower Tafenoquine was quite active in vitro in the published paper of @wwjvdsande, but was less effective in vivo than azoles or olorifim. I think nitroxoline can be administered orally, in contrast to other more toxic hydroxyquinolines such as clioquinol.

**Comment by MFernflower** [comment]
It seems nitroxoline is well tolerated orally @bebi78

**Comment by bebi78** [comment]
@MFernflower Yes, I have seen it is available as ´´Nitroxolin forte´´ capsules in Germany for the treatment of urinary tract infections.

**Comment by bebi78** [comment]
@MFernflower Nitroxoline was inactive against Mm. But your suggested 4-ethyl niclosamide derivative was active, and the ethanolamine salts of the 4SF5 and 4SCF3 analogs.

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Is there an update on the retesting of the aromachemicals from Givaudan and other aromatic liquids (eugenol derivatives, cinnamonitrile)? Should we remove their activity results from the current bioactivity excel list to show they still need to be tested? Are there plans for the active closamide ethanolamine salts and/or staurosporine, tyrphostine A9 this summer or can they be published next?

**Comment by wwjvdsande** [comment]
@bebi78 due to the end stage of @MA-Jjingyi PhD and the new PhD students still to be hired, this season will be a bit impaired in the amount of compounds we can test. We start with series 3. Tryphostine A9 also came out of another library we tested and will most likely be tested. The rest depends on how efficient the screening goes.

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi I'm planning a manuscript about the closamides together with some cancer stuff from colleagues in Kansas. Should we hurry up to write and submit it in time before Ma's PhD ends or can it published thereafter?

**Comment by MA-Jjingyi** [comment]
Hi @bebi78,  thank you for preparing the manuscript of closamides. that will be good for me to add this subject to my thesis. However, I plan to finish the thesis around June to ensure that I can finish my defense at the end of this year. I don't know when you want to publish them. If the time is later than my schedule, that's also fine. Thank you anyway and nice to work with you.

As for tyrphostin A9, it was active that IC50 is 1.3 µM. while Tyrphostin AG490 was not active. From 6000 compounds box screening, tyrphostin 9 (sf 6847) also inhibited MM55 at 2 µM. So, tyrphostine A9 and tyrphostin 9 will be tested in the upcoming larvae season. would you mind explaining the difference between these two compounds? or do you think it is necessary to test them in a chemical view? and if you also have tyrphostin 9 in your lab, would you mind sending 5mg to us, then we can start larvae testing soon?

**Comment by bebi78** [comment]
@MA-Jjingyi I suppose you will submit a cumulative thesis. So it should be no problem to have at least a manuscript draft about the closamides by June you can add to your thesis. I will ask the colleagues in Kansas City after Easter about their plan and will know more then. But I planned to mention you as first author or shared first author with the Kansas PhD student. The two active ethanolamine salts are better water-soluble than the precursor closamides and thus it might be worth testing them in vivo against larvae.

Yes, there are various names for tyrphostin A9, it is the same compound as tyrphostin 9. The inactive AG490 is a different compound. Other names for tyrphostin A9 are the mentioned SF-6847, tyrphostin AG17, GCP5126, and malonoben (refering to the malononitrile and benzylidene moieties of the compound). Its IUPAC name is [(3,5-di-tert-Butyl-4-hydroxyphenyl)methylidene]propanedinitrile. Interestingly, it is also an inhibitor of oxidative phosphorylation like niclosamide, and suppresses ATP production by leveling the proton gradient (maybe via the phenolic OH group). Both niclosamide and tyrphostin A9 were active against C. neoformans under nutrient starvation conditions (see link).

https://link.springer.com/article/10.1007/s00216-013-7134-4

**Comment by bebi78** [comment]
@MFernflower I would like to add your name to the Acknowledgments of the new Closamid (Niclosamide derivatives) manuscript. Is it OK for you?

**Comment by MFernflower** [comment]
Yes

**Comment by MFernflower** [comment]
Would it be possible for me to see the manuscript in general? @bebi78

**Comment by bebi78** [comment]
@MFernflower Yes. I would need your email address then (the manuscript also contains cancer results from a different group so I don't know if I can post it here).

**Comment by MFernflower** [comment]
@bebi78 email sent

**Comment by MFernflower** [comment]
closing this issue - Was nitroxoline ever tested for eumycetoma? @OpenSourceMycetoma/corecontrib
